mapbox_tilesets/scripts/cli.py

```diff
@@ -318,13 +325,9 @@ def jobs(tileset, stage, token=None, indent=None):
     url = "{0}/tilesets/v1/{1}/jobs?access_token={2}".format(
         mapbox_api, tileset, mapbox_token
     )
-    if stage:
-        url = "{0}/tilesets/v1/{1}/jobs?stage={2}&access_token={3}".format(
-            mapbox_api, tileset, stage, mapbox_token
-        )
-
+    url = "{0}&limit={1}".format(url, limit) if limit else url
```

Comment:
np: if we set a default for the click option of limit, we should always have a limit

Reply:
My reasoning here is that the default is set via the decorator, not with a default kwarg, so it's possible for `limit` to be `None` if you call this function directly. I'd like to keep the cli interface separate from the actual URL construction in case we change the interface